bin/run.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+usage() {
+  echo "usage: ./bin/run.sh version|lambda.zip|kinto|initialize-kinto|latest-inventory-to-kinto"
+  echo ""
+  echo "    version                     show current version"
+  echo "    lambda.zip                  build Zip archive from buildhub package"
+  echo "    kinto                       run a Kinto server"
+  echo "    initialize-kinto            initialize a Kinto server for buildhub"
+  echo "    latest-inventory-to-kinto   load latest S3 inventory to a Kinto server"
+  echo ""
+  exit 1
+}
+
+[ $# -lt 1 ] && usage
+
+source .venv/bin/activate
+
+case $1 in
+  version)
+    cat version.json
+    ;;
+  lambda.zip)
+    cd .venv/lib/python3.6/site-packages/
+    zip -r ../../../../lambda.zip *
+    ;;
+  kinto)
```

Comment:
This is confusing, is it suppose to be the action we use to run kinto in production or just locally?
If we keep kinto-dist for production, aren't we suppose to provide elasticsearch as well (kinto-elasticsearch)?

TBH, I would leave kinto and initialize-kinto out of the scope of that docker image to prevent confusion with kinto-dist.

Reply:
Yeah, I agree. 

Let's keep `initialize-kinto` though since it takes the `initialization.yml` as default parameter